index.html

```diff
@@ -149,13 +149,13 @@ <h2>Scope</h2>
             <dd>18.2% of all websites use JSON-LD.</dd>
           </dl>
 
-        <p>In the time between original publication by the RDF 1.1 Working Group and now
-          the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a>
-          has had over 100 participants and over
-          <a href="https://github.com/json-ld/json-ld.org/issues?utf8=✓&q=is%3Aissue+created%3A%3E2014-01-01+-label%3Awebsite+-label%3Aplayground+">100 issues filed</a>.
+        <p>In the time between its original publication by the RDF 1.1 Working Group, the documents have been maintained by
+          the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a> which
+          has had over 100 participants and filed around
+          <a href="https://github.com/json-ld/json-ld.org/issues?utf8=✓&q=is%3Aissue+created%3A%3E2014-01-01+-label%3Awebsite+-label%3Aplayground+">100 issues</a>.
           In response, the CG has worked to update the specifications and test-suite, accepting
           over <a href="https://github.com/json-ld/json-ld.org/pulls?utf8=✓&q=is%3Apr+created%3A%3E2014-01-01">90 pull requests</a> for a pending CG 1.1 release.
-          This issues generally relate to allowing more natural JSON idioms to be expressed using JSON-LD:
+          These issues generally relate to allow more natural JSON idioms to be expressed using JSON-LD:
```

Comment:
“Allowing” scans better to me.

Reply:
That is fine, you are the native anglo-saxon:-)

Will you change it?